### PR TITLE
Enable Content Audit allocation

### PR DESCRIPTION
**Pull request by andrewgarner**
Adds an environmental variable to enable content audit allocation in dev, integration and staging.

### CODE REVIEW

**Comment by andrewgarner** [comment]
[Add feature toggle to disable allocation in production, and enable in staging and integration](https://trello.com/c/r3iKNHcw/449-add-feature-toggle-to-disable-allocation-in-production-and-enable-in-staging-and-integration)

File: modules/govuk/manifests/apps/content_performance_manager.pp [hunk 1]
```diff
@@ -122,6 +122,14 @@
     port => $redis_port,
   }
 
+  if $::domain =~ /^.*\.(dev|integration\.publishing\.service|staging\.publishing\.service)\.gov\.uk/ {
```
Comment (alecgibson) [inline modules/govuk/manifests/apps/content_performance_manager.pp]: Isn't the more normal thing to do here to add it to the enviornment-specific config files? eg https://github.com/alphagov/govuk-puppet/blob/master/hieradata/development.yaml#L97-L183

Comment (andrewgarner) [inline modules/govuk/manifests/apps/content_performance_manager.pp]: Fair challenge. I've reworked the PR to adopt that approach 👍

File: modules/govuk/manifests/apps/content_performance_manager.pp [hunk 2]
```diff
@@ -63,6 +63,7 @@
   $db_password = undef,
   $db_username = 'content_performance_manager',
   $enable_procfile_worker = true,
+  $feature_auditing_allocation = false,
```
Comment (alecgibson) [inline modules/govuk/manifests/apps/content_performance_manager.pp]: Do we need to add a comment to the documentation at the top of this class?

Comment (andrewgarner) [inline modules/govuk/manifests/apps/content_performance_manager.pp]: Added! 👍

**Pull request merged**